src/igz_controls/components/validating-input-field/validating-input-field.component.js

```diff
@@ -18,7 +18,7 @@
      * @param {string} [compareInputValue] - If provided, the input will be valid if it matches this, or otherwise it
      *     will be invalid.
      * @param {string} [fieldType='input'] - Determines the type of field:
-     *     - `'input'` (default): a text box
+     *     - `'input'`   : a text box
```

Comment:
Instead of deleting this, please make it actually the default in `$onInit`.
If you want to go a step further see what I did with `bordersMode` binding with `BORDER_MODES` and `defaultBorderMode` variables, and you can do the same for `fieldType`.